enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackup.java

```diff
@@ -302,7 +302,7 @@ public OnlineBackup full( String targetDirectory, boolean verification, Config t
     public OnlineBackup incremental( String targetDirectory )
     {
         outcome = new BackupService().doIncrementalBackup( hostNameOrIp, port, new File( targetDirectory ),
-                timeoutMillis, defaultConfig() );
+                getConsistencyCheck( true ), timeoutMillis, defaultConfig() );
```

Comment:
Field will be upset if you change the default behavior of the old `neo4j-backup`.